Realm/RLMObjectSchema.mm

```diff
@@ -268,6 +312,7 @@ - (id)copyWithZone:(NSZone *)zone {
 
     // call property setter to reset map and primary key
     schema.properties = [[NSArray allocWithZone:zone] initWithArray:_properties copyItems:YES];
+    schema.computedProperties = [[NSArray allocWithZone:zone] initWithArray:_computedProperties copyItems:YES];
```

Comment:
`RLMProperty` has some mutable members so it seemed safest to avoid any chance of sharing them.